### DISCUSSION TITLE
Poetry is installed and added to path but other issue #584

### DISCUSSION THREAD

**Discussion by deeplow** [freedomofpress/dangerzone#585] [General]
Quoting from [issue #584](https://github.com/freedomofpress/dangerzone/issues/584):
> I have poetry installed and added to my PATH but I can't seem to find my dangerzone folder,
>
> Do I need to download something else first?

**Comment by deeplow** [comment]
@mc-gitdeb We forgot to mention the very important step of cloning the repository. Before doing `poetry install` you should

```
git clone https://github.com/freedomofpress/dangerzone/
cd dangerzone
```

**Reply by mc-gitdeb** [reply]
Oh right, I was wondering, ha, Thank you very much, much obliged, God bless you

**Comment by deeplow** [comment]
Closing this in favor of https://github.com/freedomofpress/dangerzone/discussions/586
